addons/sale/views/sale_portal_templates.xml

```diff
@@ -143,15 +143,17 @@
                     </t>
                     <t t-set="entries">
                         <div class="d-flex flex-column gap-4">
-                            <div class="d-flex flex-column gap-1 order-md-last order-lg-first" id="sale_order_sidebar_button">
+                            <div class="d-flex flex-column gap-2" id="sale_order_sidebar_button">
                                 <a t-if="sale_order._has_to_be_signed()" role="button" class="btn btn-primary d-block" data-bs-toggle="modal" data-bs-target="#modalaccept" href="#">
                                     <i class="fa fa-check"/><t t-if="sale_order._has_to_be_paid()"> Sign &amp; Pay</t><t t-else=""> Accept &amp; Sign</t>
                                 </a>
                                 <a t-elif="sale_order._has_to_be_paid()" role="button" id="o_sale_portal_paynow" data-bs-toggle="modal" data-bs-target="#modalaccept" href="#" t-att-class="'d-block %s' % ('btn btn-light' if sale_order.transaction_ids else 'btn btn-primary')" >
                                     <i class="fa fa-check"/> <t t-if="not sale_order.signature">Accept &amp; Pay</t><t t-else="">Pay Now</t>
                                 </a>
-                                <div class="o_download_pdf">
-                                    <a class="btn btn-secondary o_print_btn o_portal_invoice_print d-block" t-att-href="sale_order.get_portal_url(report_type='pdf')" id="print_invoice_report" title="View Details" target="_blank"><i class="fa fa-print"/> View Details</a>
+                                <div class="o_download_pdf d-flex gap-1 flex-lg-column flex-xl-row">
+                                    <div class="btn-group flex-grow-1">
```

Comment:
Is there a case where we see two or more buttons in this `btn-group`? If not, is this btn-group really necessary?
<img width="470" alt="Screenshot 2023-10-27 at 14 59 57" src="https://github.com/odoo/odoo/assets/19491443/eebf206b-92a6-4336-97bc-eb7945398b4a">

(same in account)